usagov/usagov-2021
# USAGOV-1857-wizard-fieldset-background-fix: Ensure the styling on the wizard fieldset element stretches the full length.

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1857

## Description

Update wizard.scss to add styling rules to the `<fieldset>` element.

## Type of Changes

- [ X ] Bugfix
- [ X ] Frontend (Twig, Sass, JS)
![image](https://github.com/user-attachments/assets/fd451cc3-b079-4fe1-b70d-a26c916d0bd5)

## Testing Instructions


1. Go to any wizard page with options.
2. Select an option.
3. The background color stretches the full length of the content area.
4. Hover over an option.
5. The background color stretches the full length of the content area.
6. Verify the previous steps in multiple viewports.

### Change Requirements



## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
7. Find the commit of this pull request.
8. Build and deploy the changes.
9. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
